tests/validation/infra/test_validate_aws_subnets.py

```diff
@@ -211,45 +209,6 @@ def test_aws_public_subnets_validation_failure(
         func(config, ec2_client)
 
 
-def test_aws_public_subnets_suffix_validation_success(
```

Comment:
were these validations removed?

Reply:
Because we are not doing suffix validations.  The validations were skipped and not needed.  If we need to add them back later we can.